src/components/Carousel/Carousel.styles.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import styled from "styled-components";
+import { Icon } from "../Icon/Icon";
+
+export const Wrapper = styled.div`
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+`;
+
+export const SlidesContainer = styled.div``;
+
+export const Slides = styled.ul``;
+
+export const Card = styled.li`
+  display: flex;
+  width: fit-content;
+`;
+
+export const Button = styled.button`
+  display: flex;
+  backdrop-filter: blur(40px);
+  color: ${({ theme }) => theme.colors.black};
+  background-color: #ffffff70;
+  height: 2rem;
+  width: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: 300ms;
+
```

Comment:
```suggestion
  box-shadow: 0 0.2rem 0.4rem ${({ theme }) => theme.colors.gray};
```
I thought the button was barely noticeable, maybe a box-shadow like this or something similar could make it stand out a bit more